lib/pavilion/series/test_set.py

```diff
@@ -255,6 +257,9 @@ def make(self, build_only=False, rebuild=False, local_builds_only=False,
         # make sure result parsers are ok
         self.check_result_format(self.tests)
 
+        for test in skip_tests:
```

Comment:
print number of tests, number of tests skipped, and status.path to outfile instead